rbi/core/kernel.rbi

```diff
@@ -451,33 +451,33 @@ module Kernel
     params(
         initial: T.any(Integer, Float, Rational, BigDecimal, String),
         digits: Integer,
+        exception: T::Boolean
     )
     .returns(BigDecimal)
   end
-  def BigDecimal(initial, digits=0); end
+  def BigDecimal(initial, digits=0, exception: true); end
+  # As of 2.6, all these numeric conversion methods *can* return `nil` iff given the kwarg `exception: false`, which
+  # means the non-nilable return value is technically incorrect, but it seemed like the least worst option given that
+  # 1) usage of the exception kwarg is uncommon, and 2) Sorbet doesn't allow us to overload sigs if methods have kwargs
 
   sig do
     params(
         x: T.any(Numeric, String),
         y: T.any(Numeric, String),
+        exception: T::Boolean
     )
     .returns(Complex)
   end
-  sig do
-    params(
-        x: String,
-    )
-    .returns(Complex)
-  end
-  def Complex(x, y=T.unsafe(nil)); end
+  def Complex(x, y=T.unsafe(nil), exception: false); end
```

Comment:
could you please duplicate the comment in all of them?

Reply:
Duplicated!